chain-indexer/src/application.rs

```diff
@@ -336,6 +336,16 @@ async fn index_block(
         );
     }
 
+    // Handle genesis block: extract any pre-funded unshielded UTXOs
```

Comment:
Add trailing dot (please start memoizing this, I am adding this comment in too many reviews).